DeploymentExperiences.md

```diff
@@ -0,0 +1,12 @@
+# Summarizing how the process went for you, what went well and what was tricky, and how far you got
+
+1. Install new dependencies 
+    - Even though I installed all the dependencies, VS code pilint showed me 'unablie to import decouple'. I don't know how to fix this lint error.
```

Comment:
Where you able to get this fixed? Chances are you either need to `pipenv install` or `pipenv shell`.